wiki/Guides/Livestreaming_osu!/en.md

```diff
@@ -10,9 +10,9 @@ Now you will need to setup your account for livestreaming osu!. From the main pa
 
 ![](img/dashboard.png "Entering the Creator Dashboard")
 
-Creator Dashboard is the place where you can configure your stream information, chat moderation, profile info, and more. For now, we will focus on stream information. From the main page of the dashboard, click on the `Stream Manager` button on the left, and then choose the `Edit Stream Info` option on the right.
+In the Creator Dashboard, you can configure stream information, chat moderation, and profile info, among other customizations. Most important to getting started is configuring your stream's information. From the main page of the dashboard, click on the `Stream Manager` button on the left, and then choose the `Edit Stream Info` option on the right.
```

Comment:
british english

```suggestion
In the Creator Dashboard, you can configure stream information, chat moderation, and profile info, among other customisations. Most important to getting started is configuring the stream information. From the main page of the dashboard, click on the `Stream Manager` button on the left, and then choose the `Edit Stream Info` option on the right.
```

Reply:
_In the Creator Dashboard, you can configure stream information, chat moderation, and profile info, among other customizations._

If you are using a comma I think "and" isn't necessary?

Reply:
to me it reads better

Reply:
Thanks, added.